spec/lib/vmdb/configuration_encoder_spec.rb

```diff
@@ -44,44 +44,50 @@
     end
   end
 
-  VMDB_CONFIG_STRINGS = <<-YAML
+  let(:vmdb_config_strings) do <-YAML
   one:
     two: two
     three:
       four: four
   YAML
+  end
+
 
-  VMDB_CONFIG_SYMBOLIZED = <<-YAML
+  let(:vmdb_config_symbolized) do <-YAML
```

Comment:
I find the `<-YAML` on the same line weird...it probably should be moved to the next line.